hazelcast/src/test/java/com/hazelcast/map/EvictionTest.java

```diff
@@ -443,20 +423,21 @@ protected void testEvictionLFUInternal(boolean disableStats) {
 
     @Test
     public void testEvictionLFU2() {
-        final int size = 10000;
-        final String mapName = randomMapName("testEvictionLFU2");
-
-        Config cfg = getConfig();
-        cfg.setProperty(GroupProperty.PARTITION_COUNT.getName(), "1");
-        MapConfig mc = cfg.getMapConfig(mapName);
-        mc.setEvictionPolicy(EvictionPolicy.LFU);
-        mc.setEvictionPercentage(90);
-        MaxSizeConfig msc = new MaxSizeConfig();
-        msc.setMaxSizePolicy(MaxSizeConfig.MaxSizePolicy.PER_NODE);
-        msc.setSize(size);
-        mc.setMaxSizeConfig(msc);
-
-        HazelcastInstance node = createHazelcastInstance();
```

Comment:
A bit hard to spot after all the cleanups, but here was the `createHazelcastInstance()` call which didn't use the created `Config`.